files/templates/scrapeconfigs/additional-scrape-configs.template.yaml

```diff
@@ -500,42 +502,23 @@
 # node-exporter
 - job_name: [[ .ClusterID ]]-prometheus/node-exporter-[[ .ClusterID ]]/0
   honor_labels: true
-  scheme: http
+  scheme: https
   kubernetes_sd_configs:
-  - role: endpoints
-    namespaces:
-      names:
-[[ if eq .ClusterType "management_cluster" ]]
-      - monitoring
-[[ else ]]
-      - kube-system
-[[ end ]]
+  - role: pod
 [[ include "_apiserver" . ]]
 [[ include "_tlsconfig_skip" . | indent 2 ]]
   relabel_configs:
-  - source_labels: [__meta_kubernetes_service_label_app]
-    regex: node-exporter
+  - target_label: __address__
+    replacement: [[ .APIServerURL ]]
+  - source_labels: [__meta_kubernetes_pod_name]
+    regex: (node-exporter.*)
+    target_label: __metrics_path__
+    replacement: /api/v1/namespaces/[[ include "_node_exporter_namespace" . ]]/pods/${1}:10300/proxy/metrics
+  - source_labels: [__meta_kubernetes_namespace, __meta_kubernetes_pod_name]
+    regex: [[ include "_node_exporter_namespace" . ]];node-exporter.*
     action: keep
-  - source_labels: [__meta_kubernetes_service_label_app]
+  - source_labels: [__meta_kubernetes_pod_container_name]
     target_label: app
-  - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_path]
-    regex: (.+)
-    target_label: __metrics_path__
-    replacement: $1
-    action: replace
-  - source_labels: [__address__, __meta_kubernetes_service_annotation_prometheus_io_port]
-    regex: (.+)(?::\d+);(\d+)
-    target_label: __address__
-    replacement: $1:$2
-    action: replace
-  - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_scheme]
-    regex: (https?)
-    target_label: __scheme__
-    replacement: $1
-    action: replace
-  - source_labels: [__meta_kubernetes_service_annotationpresent_giantswarm_io_monitoring, __meta_kubernetes_service_labelpresent_giantswarm_io_monitoring]
-    regex: .*(true).*
-    action: drop
```

Comment:
I reckon this is kind of important otherwise we get duplicate targets from node-exporter app which is using the `giantswarm.io/monitoring` annotation.